orc8r/cloud/deploy/terraform/orc8r-aws/db.tf

```diff
@@ -26,8 +26,29 @@ resource "aws_db_instance" "default" {
 
   db_subnet_group_name = module.vpc.database_subnet_group
 
+  backup_retention_period = var.orc8r_db_backup_retention
+  backup_window           = var.orc8r_db_backup_window
+
   skip_final_snapshot = true
   # we only need this as a placeholder value for `terraform destroy` to work,
   # this won't actually create a final snapshot on destroy
   final_snapshot_identifier = "foo"
 }
+
+resource "aws_sns_topic" "sns_orc8r_topic" {
```

Comment:
Can you add a boolean(aws_db_event_alerts) which is optional and turned off by default, which can gate these other variables. These lack of this mandatory variable(orc8r_sns_email) may cause errors in existing upgrades and will lead to unnecessary confusion.

Reply:
Can I set the boolean gate just for the mandatory variable (orc8r_sns_email)? The SNS and the event subscription would exist based on default values enabled in the configurations, but no notification would be sent unless the customer desires so. 

I would like to keep the backup option enabled to ensure we are protected against EC2 instance failure coupled with an irrecoverable EBS volume. WDYT?